server/routers/students.js

```diff
@@ -39,4 +39,16 @@ router.postAsync('/api/students/create', async (req, res) => {
   });
 });
 
+router.deleteAsync('/api/students/delete/:id', async (req, res) => {
+  const [err, students] = await go(Student.find({}));
+
+  Student.findOneAndDelete({studentId: req.params.id}, (err, student) => {
```

Comment:
delete by setting `timeDeleted` to `Date.now()` - also, can you change the `/api/students` route to only grab students that get students where `timeDeleted` is not null?

Reply:
also use async await :)